src/helpers/figures/charts/pie_chart.ts

```diff
@@ -212,6 +213,23 @@ function getPieConfiguration(
   config.options.plugins!.tooltip!.callbacks!.title = function (tooltipItems) {
     return tooltipItems[0].dataset.label;
   };
+  config.options.plugins!.tooltip!.callbacks!.label = function (tooltipItems) {
+    const { format, locale } = localeFormat;
+    const data = tooltipItems.chart.data;
+    const percentageValues = calculatePercentageValues(data.datasets);
+    const dataIndex = tooltipItems.dataIndex;
+    const datasetIndex = tooltipItems.datasetIndex;
+    const percentage = percentageValues[datasetIndex || 0][dataIndex!];
+
+    let xLabel = tooltipItems.label;
+    const yLabel = tooltipItems.parsed.y ?? tooltipItems.parsed;
+    const yLabelStr =
+      format && typeof yLabel === "number"
+        ? formatValue(yLabel, { format, locale })
+        : yLabel?.toLocaleString() || "";
```

Comment:
Don't use `toLocaleString`. You can use an undefined format and non number values with `formatValue`.